src/BenchmarkDotNet.Diagnostics.Windows/Sessions.cs

```diff
@@ -21,7 +21,7 @@ public HeapSession(DiagnoserActionParameters details, EtwProfilerConfig config,
         {
         }
 
-        protected override string FileExtension => ".userheap.etl";
+        protected override string FileExtension => "userheap.etl";
```

Comment:
nice catch! I wonder why we have not found the double dots `..` in the file names before ;p

Reply:
;) I've broken it, so I'm fixing it.